crates/util/src/command.rs

```diff
@@ -0,0 +1,32 @@
+use std::ffi::OsStr;
+
+#[cfg(target_os = "windows")]
+const CREATE_NO_WINDOW: u32 = 134217728u32;
```

Comment:
```suggestion
const CREATE_NO_WINDOW: u32 = 0x08000000;
```

Small nitpick, but I believe this to be the more readable and understandable representation for the flag.

Reply:
Thx!

Reply:
No worries, thank you for all your awesome contributions for the community!